### ISSUE TITLE
RT GUIClips Spam

### ISSUE THREAD

**Issue by KSP-TaxiService** [KSP-TaxiService/RemoteTech#1]
Stock KSP 1.2.1586

Reproduction steps:
1) Get into KSC of a sandbox save
2) Alt + F12 to bring up the RT debug window (other than KSP cmd window)
3) Click the third button of "GUID-Reader"
4) Spam messages on the GUIClips error

Relevant error:
GUI Error: You are pushing more GUIClips than you are popping. Make sure they are balanced)

(Filename:  Line: 422)

NullReferenceException: Object reference not set to an instance of an object
  at RemoteTech.UI.DebugWindow.drawGuidReader () [0x00000] in <filename unknown>:0 
  at RemoteTech.UI.DebugWindow.Window (Int32 uid) [0x00000] in <filename unknown>:0 
  at RemoteTech.UI.AbstractWindow.WindowPre (Int32 uid) [0x00000] in <filename unknown>:0 
  at UnityEngine.GUILayout+LayoutedWindow.DoWindow (Int32 windowID) [0x00000] in <filename unknown>:0 
  at UnityEngine.GUI.CallWindowDelegate (UnityEngine.WindowFunction func, Int32 id, UnityEngine.GUISkin _skin, Int32 forceRect, Single width, Single height, UnityEngine.GUIStyle style) [0x00000] in <filename unknown>:0 

(Filename:  Line: -1)

GUI Error: You are pushing more GUIClips than you are popping. Make sure they are balanced)

**Comment by KSP-TaxiService** [comment]
More details:
I narrowed to the null nature of RTCore.Instance that is not passed/initialised in DebugWindow class. I am trying to find out how RTCore.Instance is initialised in the first place.

With the debugging environment set up with the assistance of neitsa and one person (fotgot name atm), I observed that DebugWindow tried to call RTCore.Instance at the KSC scene, and RTCore.Instance for KSC scene calls the RTSpaceCentre class that lacks of NetworkManager object.
